static/open-specs/liveobjects.yaml

```diff
@@ -0,0 +1,641 @@
+                  - id: "myIdempotencyKey:1"
+                    operation: "COUNTER_INC"
+                    objectId: "counter:DXr2i8FHRGkLrHPccWhXKDj1VUX2s7ACvmT..."
+                    data:
+                      number: 1
+              createMap:
+                summary: Create a map
+                value:
+                  operation: MAP_CREATE
+                  data:
+                    myMapKey: { string: "myMapValue" }
+                    myOtherKey: { boolean: true }
+              setValueOnMap:
+                summary: Set a value on a map
+                value:
+                  operation: MAP_SET
+                  objectId: "map:99OSLwCFQsrs6GID4M8rfO_0sYmVwRADcE..."
+                  data:
+                    key: "foo"
+                    value: { string: "bar" }
+              removeValueFromMap:
+                summary: Remove a value from a map
+                value:
+                  operation: MAP_REMOVE
+                  objectId: "map:99OSLwCFQsrs6GID4M8rfO_0sYmVwRADcECua_@174220..."
+                  data:
+                    key: "myMapKey"
+              createCounter:
+                summary: Create a counter
+                value:
+                  operation: COUNTER_CREATE
+                  data:
+                    number: 3.1415926
+              incrementCounter:
+                summary: Increment a counter
+                value:
+                  operation: COUNTER_INC
+                  objectId: "counter:J7x6mAF8X5Ha60VBZb6GtXSgnKJQagNLgadUlg..."
+                  data:
+                    number: 2
+      responses:
+        '200':
+          description: Successfully created or updated objects.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  channel:
+                    type: string
+                    description: The channel ID.
+                    example: "<channelId>"
+                  objectIds:
+                    type: array
+                    items:
+                      type: string
+                    description: The list of object IDs created or updated.
+                    example: ["map:YIffJYRAP2k2e7ZP+xzequ9c5kDu1LfI/sEOKoWHvv4@1742479683000"]
+        '400':
+          description: Invalid request.
+  /channels/{channelId}/objects/{objectId}:
+    get:
+      summary: Gets an object using an object ID.
+      description: |
+        Fetch a live object stored on the channel in a tree structure.
+      tags:
+      parameters:
+        - name: channelId
+          in: path
+          required: true
+          description: The channel ID.
+          schema:
+            type: string
+            example: "my-app:likes:counter"
+        - name: objectId
+          in: path
+          required: true
+          description: The object ID.
+          schema:
+            type: string
+            example: "object-123"
```

Comment:
I think here we need to say that the default that customer can use out of the box is "root"